src/data/languages/Spanish.json

```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],

```

Comment:
Question: Would it make sense to have Portuguese as a fallback before English? That is, if there's a Spanish key missing, the script will first see if there's a Portuguese one before using the English key.

Reply:
The syntax would be ``"fallback": ["Portuguese", "English"],``

Reply:
It is better in English as Portuguese is not widely spoken in Latin America.